### DISCUSSION TITLE
MIL-STD 1472

### DISCUSSION THREAD

**Discussion by markacianfrani** [RocketCommunicationsInc/astro#766] [Announcements]
We are currently evaluating Astro’s compliance with MIL-STD 1472 Section 15. We are in the process of publishing our initial findings on https://astrouxds.com/compliance/mil-std-1472. This discussion area is provided to foster discussion and feedback about Astro’s adherence to MIL-STD 1472.

**Comment by ShaunDougherty** [comment]
MIL-STD 1472 H is really revolving around human factors, how much research/analysis have you done with the compliance to MIL-STD 2525D?  I ask this because 2525 is much more detailed driven to the design of gov interface design.

**Reply by ShaunDougherty** [reply]
Also have you done any compliance analysis yet  to USWDS and 508  (accessibility standards)?

**Comment by dmcalester** [comment]
Hi Shaun.

So in our review of 2525D it seemed more targeted at iconography and symbology. We did – and continue to – account for its guidance on color and luminosity for our status colors. If you have any thoughts on specific sections or content we should pay attention we’re definitely interested. To date our users have expressed the need for 1472 Compliance and/or waiver language they can use, we have not received similar requests for 2525D so it’s been a lower priority.

USWDS serves as an inspiration for us, but as they are more focused on public-facing web sites and Astro is focused on enterprise applications we have quite different use cases, we use what we can though. To date we have had very little interest in an Astro for web sites, but if the community identifies a need it end up on the road map.

508 is in our backlog, but not part of our current PI objectives. Right now SMC-S-023 Vol 1 and Vol 2 assessments are a priority given our customer is SSC (née SMC). 

Out of curiosity can you share what program(s) you’re working in? As you can imagine it’s a challenge to know who and how open source software is being utilized. If it’s not something you can share comfortably, please feel free to reach out via email: duncan@rocketcom.com

**Reply by cheapwebmonkey** [reply]
I would love to utilize this system for some of the work we are doing, but the lack of accessibility/508 compliance and support for responsive/mobile browsers is a deal-breaker for our use cases. 

Reading that 508 is in the backlog and not part of PI is disappointing, as internal apps and software need to be as accessible as external digital products are.
